_internal/quarto/.gitignore

```diff
@@ -0,0 +1 @@
+/.quarto/
```

Comment:
can you please extend that list by quarto specific dirs and files like `_site/`, `_freeze/`, `*_cache/` and others

Reply:
I see the `_site` in the main `.gitignore` but it makes sense to have it here to be explicit.
I'm not aware of all possible autogenerated directories that quarto can create, Is there some place where I can refer?
For now I can place these:
```
_site
_book
_freeze/
*_cache/
```

Reply:
I'll refer to the .gitignore from falcon. It has a bunch of directories to ignore:
```
quarto/.quarto/
quarto/site_libs/
/.quarto/
_site
_freeze
_cache
cache
*_cache
*_data
*_files
figures
```